src/rt/profilegc.d

```diff
@@ -21,7 +21,7 @@ import core.stdc.string;
 
 import core.exception : onOutOfMemoryError;
 
-struct Entry { size_t count, size; }
+struct Entry { ulong count, size; }
```

Comment:
Why is changing `size_t` to `ulong` is necessary or better? Needs clarification in the commit message at least, but maybe even here too, so nobody is tempted to change it back.